lib/orangeade/generator/boolean.ex

```diff
@@ -0,0 +1,33 @@
+defmodule Orangeade.Generator.Boolean do
+  @moduledoc """
+  A boolean values generator {:true, :false, :nil}
+  
+  
+  ## Examples
+
+      iex(1)> Caffeine.Stream.take(
+      ...(1)> Orangeade.Generator.Boolean.stream(),
+      ...(1)> 5_000)
+      [false, false, nil, nil, nil, nil, false, true, false, nil, nil, nil, nil, true,
+      true, nil, nil, nil, nil, false, nil, false, nil, false, nil, false, true,
+      false, true, false, true, false, nil, false, true, false, false, false, false,
+      false, false, true, nil, false, false, false, false, nil, true, true, ...]
+
+  """
+  def stream do
+    alias Orangeade.Generator.BoundNatural, as: BoundNatural
+    BoundNatural.stream(limit: 3)
```

Comment:
I made a similar comment to Robert about piping as I would here... you decide what you consider better style.